README.md

```diff
@@ -13,16 +13,16 @@
 ## Step 2/4: Run image
 
     # Ye, I known `xhost +` is a bad practice but ¯\_(ツ)_/¯
-    xhost +
+    xhost + $(ifconfig en0 | grep inet | awk '$1=="inet" {print $2}')
     docker run \
-        -ti \
-        --env DISPLAY=$DISPLAY \
-        --network=host \
+        -it \
+        --env DISPLAY=$(ifconfig en0 | grep inet | awk '$1=="inet" {print $2}'):0 \
         --name nostale \
         --privileged \
+        --rm \
```

Comment:
Don't rm your image or you will have to reinstall Nostale

Reply:
Caching by binding the $HOME/.wine is fine

Reply:
Oh, you're right but don't cache all `wine` directory. Only the Nostale directory would be better, I think.

Reply:
Nostale need Gecko, Mono and other dependencies to work so we need everything

Reply:
You can press `cancel` when `wine` asks you to install Gecko and Mono, it will works

Reply:
I cached only nostale